http_service/app.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import copy
+import os
+
+from flask import Flask, g, jsonify
+
+from bugbug import bugzilla
+
+from .models import load_model
+
+application = Flask(__name__)
+
+bugzilla.set_token(os.environ.get("BUGBUG_BUGZILLA_TOKEN"))
+
+
+def get_model(model):
+    attribute = f"bugbug_model_{model}"
+
+    if not hasattr(g, attribute):
+        model = load_model(model)
+        setattr(g, attribute, model)
+
+    return getattr(g, attribute, model)
```

Comment:
I'm not familiar with `g`. Will this cache the model between requests?
According to http://flask.pocoo.org/docs/1.0/appcontext/#storing-data it doesn't, so maybe we should use something else so we avoid loading the model on each request?

Reply:
Indeed, I was remembering incorrectly that it was surviving requests, I will store the models somewhere else.